cypress/integration/styling-and-classes.spec.ts

```diff
@@ -104,4 +104,13 @@ describe('Styling and Classes', () => {
         });
     });
   });
+
+  it(`should add '.dts-disabled' class to disabled items`, () => {
```

Comment:
Just say `'dts-disabled'`, that's what I did for all the other tests as well.
```suggestion
  it(`should add 'dts-disabled' class to disabled items`, () => {
```